[2025-06-19T09:27:43.357+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-19T09:27:43.390+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nba_etl_to_postgres.fetch_shot_chart_and_load_to_postgres manual__2025-06-19T09:16:58.571124+00:00 [queued]>
[2025-06-19T09:27:43.550+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nba_etl_to_postgres.fetch_shot_chart_and_load_to_postgres manual__2025-06-19T09:16:58.571124+00:00 [queued]>
[2025-06-19T09:27:43.551+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-06-19T09:27:43.603+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): fetch_shot_chart_and_load_to_postgres> on 2025-06-19 09:16:58.571124+00:00
[2025-06-19T09:27:43.610+0000] {standard_task_runner.py:63} INFO - Started process 1177 to run task
[2025-06-19T09:27:43.614+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nba_etl_to_postgres', 'fetch_shot_chart_and_load_to_postgres', 'manual__2025-06-19T09:16:58.571124+00:00', '--job-id', '82', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmplaaqa960']
[2025-06-19T09:27:43.615+0000] {standard_task_runner.py:91} INFO - Job 82: Subtask fetch_shot_chart_and_load_to_postgres
[2025-06-19T09:27:43.702+0000] {task_command.py:426} INFO - Running <TaskInstance: nba_etl_to_postgres.fetch_shot_chart_and_load_to_postgres manual__2025-06-19T09:16:58.571124+00:00 [running]> on host 11d91b3c2b82
[2025-06-19T09:27:43.847+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='nba_etl_to_postgres' AIRFLOW_CTX_TASK_ID='fetch_shot_chart_and_load_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2025-06-19T09:16:58.571124+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-19T09:16:58.571124+00:00'
[2025-06-19T09:27:43.848+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-19T09:27:43.872+0000] {logging_mixin.py:188} WARNING - /opt/***/dags/etl_pipeline.py:158 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-06-19T09:27:48.248+0000] {logging_mixin.py:188} INFO - Processed game on 2024-10-22 between BOS and NYK.
[2025-06-19T09:27:55.224+0000] {logging_mixin.py:188} INFO - Processed game on 2024-10-22 between LAL and MIN.
[2025-06-19T09:28:02.453+0000] {logging_mixin.py:188} INFO - Processed game on 2024-10-23 between DET and IND.
[2025-06-19T09:28:08.303+0000] {logging_mixin.py:188} INFO - Processed game on 2024-10-23 between ATL and BRK.
[2025-06-19T09:28:15.134+0000] {logging_mixin.py:188} INFO - Processed game on 2024-10-23 between MIA and ORL.
[2025-06-19T09:28:22.828+0000] {logging_mixin.py:188} INFO - Processed game on 2024-10-23 between PHI and MIL.
[2025-06-19T09:28:29.746+0000] {logging_mixin.py:188} INFO - Processed game on 2024-10-23 between TOR and CLE.
[2025-06-19T09:28:35.835+0000] {logging_mixin.py:188} INFO - Processed game on 2024-10-23 between HOU and CHO.
[2025-06-19T09:28:42.799+0000] {logging_mixin.py:188} INFO - Processed game on 2024-10-23 between NOP and CHI.
[2025-06-19T09:28:49.642+0000] {logging_mixin.py:188} INFO - Processed game on 2024-10-23 between UTA and MEM.
[2025-06-19T09:28:56.579+0000] {logging_mixin.py:188} INFO - Processed game on 2024-10-23 between LAC and PHO.
[2025-06-19T09:29:02.517+0000] {logging_mixin.py:188} INFO - Processed game on 2024-10-23 between POR and GSW.
[2025-06-19T09:29:09.615+0000] {logging_mixin.py:188} INFO - Processed game on 2024-10-24 between WAS and BOS.
[2025-06-19T09:29:16.650+0000] {logging_mixin.py:188} INFO - Processed game on 2024-10-24 between DAL and SAS.
[2025-06-19T09:29:23.680+0000] {logging_mixin.py:188} INFO - Processed game on 2024-10-24 between DEN and OKC.
[2025-06-19T09:29:29.455+0000] {logging_mixin.py:188} INFO - Processed game on 2024-10-24 between SAC and MIN.
[2025-06-19T09:29:36.778+0000] {logging_mixin.py:188} INFO - Processed game on 2024-10-25 between ORL and BRK.
[2025-06-19T09:29:43.818+0000] {logging_mixin.py:188} INFO - Processed game on 2024-10-25 between TOR and PHI.
[2025-06-19T09:29:50.698+0000] {logging_mixin.py:188} INFO - Processed game on 2024-10-25 between ATL and CHO.
[2025-06-19T09:29:57.799+0000] {logging_mixin.py:188} INFO - Processed game on 2024-10-25 between CLE and DET.
[2025-06-19T09:29:57.807+0000] {logging_mixin.py:188} INFO - Ensuring table 'stgshotchart' exists...
[2025-06-19T09:29:57.809+0000] {logging_mixin.py:188} INFO - Table 'stgshotchart' is ready.
[2025-06-19T09:29:57.809+0000] {logging_mixin.py:188} INFO - Truncating table stgshotchart...
[2025-06-19T09:29:57.816+0000] {logging_mixin.py:188} INFO - Loading 3507 rows into stgshotchart...
[2025-06-19T09:29:57.887+0000] {logging_mixin.py:188} INFO - Database execution error: integer out of range
[2025-06-19T09:29:57.888+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-19T09:29:57.889+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/etl_pipeline.py", line 181, in fetch_and_load_shot_chart_to_stage
    load_df_to_postgres(df=final_df, table_name=table_name, conn_string=conn_string, create_table_query=create_table_query)
  File "/opt/airflow/dags/etl_pipeline.py", line 66, in load_df_to_postgres
    execute_values(cursor, sql_query, data_tuples)
  File "/home/airflow/.local/lib/python3.9/site-packages/psycopg2/extras.py", line 1299, in execute_values
    cur.execute(b''.join(parts))
psycopg2.errors.NumericValueOutOfRange: integer out of range

[2025-06-19T09:29:57.953+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=nba_etl_to_postgres, task_id=fetch_shot_chart_and_load_to_postgres, run_id=manual__2025-06-19T09:16:58.571124+00:00, execution_date=20250619T091658, start_date=20250619T092743, end_date=20250619T092957
[2025-06-19T09:29:58.012+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 82 for task fetch_shot_chart_and_load_to_postgres (integer out of range
; 1177)
[2025-06-19T09:29:58.036+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-06-19T09:29:58.060+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-19T09:29:58.089+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
